packages/core/src/lib/services/wallet-modules/wallet-modules.service.ts

```diff
@@ -283,8 +283,9 @@ export class WalletModules {
         continue;
       }
 
+      // Skip duplicated module.
       if (modules.some((x) => x.id === module.id)) {
-        throw new Error("Duplicate module id detected: " + module.id);
+        continue;
```

Comment:
Curious as to why this was removed - should we not still be alerting someone if they configure the selector with multiple instances of the same module -- unique `id` values are important because they are used as keys on an object

Reply:
If we add 2 wallets with the same id the last one this way will be skipped.
 If we don't add `continue` here the initialization of the wallet selector in `setupWalletSelector` will fail with the above error.
This was done to prevent failing of `setupWalletSelector` when we want to override one of the `default wallets`.

Reply:
Developers may want to override one of the default wallets in the list to add custom options in the setup function of that specific wallet.